### PR TITLE
New test created for the 'line' and 'paragraph' methods of lorem

### PR DESCRIPTION
**Pull request by raulovidiu**
@wlsf82 , The important change is a new test scenario added to use "line" and "paragraphs" methods of the "lorem" namespace when Creating a New Blog Post.

There are some trivial ones as well, such as fixing an errata in Readme file and the refactorization of Create Blog Post to use the Page Object Pattern.

### CODE REVIEW

File: protractor.conf.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
-'use strict';
+"use strict";
```
Comment (wlsf82) [inline protractor.conf.js]: Why not using simple quotes here?

File: spec.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
-'use strict';
+"use strict";
+
+const faker          = require('faker');
```
Comment (wlsf82) [inline spec.js]: This extras spaces are not recommended, as per Clean Code's book.

File: spec.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+const CreateBlogPost = require('./page-objects/create-blog-post.pageObject.js');
 
-const faker = require('faker');
 
 describe('Choko - Create account', () => {
-  it('password not match', () => {
+  xit('password not match', () => {
```
Comment (wlsf82) [inline spec.js]: Why you are skipping this test?

File: page-Objects/create-blog-post.pageObject.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+	constructor() {
+		this.name = element(by.id('element-type-blog-name'));
+		this.title = element(by.id('element-type-blog-title'));
+		this.body = element(by.className('note-editable'));
+		this.save = element(by.id('element-type-blog-submit'));
+	}
+
+};
+
```
Comment (wlsf82) [inline page-Objects/create-blog-post.pageObject.js]: I recommend that the directory name is:
* page-objects
or
* pageObjects
but not
page-Objects

File: page-Objects/create-blog-post.pageObject.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+class CreateBlogPost {
+
+	constructor() {
```
Comment (wlsf82) [inline page-Objects/create-blog-post.pageObject.js]: The code from the actual repo uses two spaces when using TAB, would be nice to maintain the standard.

File: protractor.conf.js [hunk 6]
```diff
@@ -1,8 +1,14 @@
...
 module.exports.config = {
   directConnect: true,
+
   baseUrl: 'http://choko.org/',
-  specs: [ 'spec.js' ],
+
+  specs: ['spec.js'],
+
   capabilities: { 'browserName': 'chrome' },
+  onPrepare() {
```
Comment (wlsf82) [inline protractor.conf.js]: You added extra spaces between each configuration, but forgot adding it here.

File: page-Objects/create-blog-post.pageObject.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+"use strict";
```
Comment (wlsf82) [inline page-Objects/create-blog-post.pageObject.js]: Why not using simple quotes here?

File: spec.js [hunk 8]
```diff
@@ -1,9 +1,11 @@
-'use strict';
+"use strict";
```
Comment (wlsf82) [inline spec.js]: Why not using simple quotes here?

File: page-Objects/create-blog-post.pageObject.js (hunk 4 above)
Comment (wlsf82) [inline page-Objects/create-blog-post.pageObject.js]: Also, I would recommend that the page object file name could be:
createBlogPost.po.js

File: spec.js [hunk 9]
```diff
@@ -23,17 +25,50 @@ describe('Choko - Create account', () => {
 });
 
 describe('Choko - Blog', () => {
-  it('try to create blog post without permission', () => {
-    const ramdonName = faker.lorem.word();
-    const randomTitle = faker.lorem.words();
-    const randomText = faker.lorem.sentences();
+  // instantiating a new instance of CreateBlogPost
```
Comment (wlsf82) [inline spec.js]: I think this comment is not necessary. Reading the code is code enough for understanding what it is doing.

File: spec.js [hunk 10]
```diff
@@ -23,17 +25,50 @@ describe('Choko - Create account', () => {
...
+  const createBlogPost = new CreateBlogPost();
+
+  // defining references to faker and exposing those throughout the "Choko - Blog" describe block
```
Comment (wlsf82) [inline spec.js]: Do you really think this is a necessary comment? I always prefer readable code than code with comments.

File: spec.js [hunk 11]
```diff
@@ -23,17 +25,50 @@ describe('Choko - Create account', () => {
...
+  const ramdonName = faker.lorem.word();
+  const randomTitle = faker.random.words();
+  // creating a reference for the lorem namespace
+  const randomText = faker.lorem;
+
+  xit('try to create blog post without permission', () => {
```
Comment (wlsf82) [inline spec.js]: Why is this being skipped?

File: spec.js [hunk 12]
```diff
@@ -23,17 +25,50 @@ describe('Choko - Create account', () => {
...
-    const ramdonName = faker.lorem.word();
-    const randomTitle = faker.lorem.words();
-    const randomText = faker.lorem.sentences();
+  // instantiating a new instance of CreateBlogPost
+  const createBlogPost = new CreateBlogPost();
+
+  // defining references to faker and exposing those throughout the "Choko - Blog" describe block
+  const ramdonName = faker.lorem.word();
+  const randomTitle = faker.random.words();
+  // creating a reference for the lorem namespace
```
Comment (wlsf82) [inline spec.js]: Again, I don't think the comment is necessary.

File: protractor.conf.js (hunk 1 above)
Comment (raulovidiu) [inline protractor.conf.js]: I'm just following the Mozilla DEV Network docs.

**Pull request closed without merging**